src/fides/api/ctl/sql_models.py

```diff
@@ -353,6 +381,49 @@ def get_data_uses(self, include_parents: bool = True) -> set[str]:
         return self.collapse_data_uses(self.privacy_declarations or [], include_parents)
 
 
+class PrivacyDeclaration(Base):
+    """
+    The SQL model for a Privacy Declaration associated with a given System.
+    """
+
+    name = Column(String)  # labeled as Processing Activity in the UI
+    ### keep egress/ingress as JSON blobs as they have always been
+    egress = Column(JSON)
+    ingress = Column(JSON)
+
+    ### references to other tables, but kept as 'soft reference' strings for now
+    data_categories = Column(ARRAY(String))
+    data_qualifier = Column(String)
+    data_subjects = Column(ARRAY(String))
+    dataset_references = Column(ARRAY(String))
+
+    ### proper FK references to other tables
+    # System
+    system_id = Column(
+        String,
+        ForeignKey(System.fides_key, ondelete="CASCADE", onupdate="CASCADE"),
+        nullable=False,
+        index=True,
+    )
+    system = relationship(System, back_populates="privacy_declarations")
+
+    # DataUse
+    # the FK column is just plain `data_use` to align with the `FidesKey` field on the pydantic model
+    data_use = Column(String, ForeignKey(DataUse.fides_key), nullable=False, index=True)
```

Comment:
perhaps - but i think that would lead to some changed behavior, and just want to confirm that's desirable, per [comment here](https://github.com/ethyca/fides/pull/3098#discussion_r1172940489)

Reply:
I'm talking about cascade update here specifically, not delete.  A concern that I have with another FK on a non-PK field.

Reply:
ah OK - that makes sense. i think i'll be removing the FK reference here generally per the other conversation (a bit of a bummer, but i understand why we want to do that) -- so i believe this will be moot.